src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.WorkerThread.cs

```diff
@@ -232,6 +232,7 @@ private static bool TryCreateWorkerThread()
                     RuntimeThread workerThread = RuntimeThread.Create(WorkerThreadStart);
                     workerThread.IsThreadPoolThread = true;
                     workerThread.IsBackground = true;
+                    workerThread.Name = $"Worker Thread {workerThread.ManagedThreadId}";
```

Comment:
Looks like CoreCLR is not setting the name (it's just null). Seems unnecessary since there are properties to determine all of the information provided here.